dist/index.js

```diff
@@ -110,7 +110,7 @@ async function run () {
 
       // final bits of the install
       await execute('mkdir -p ' + path.join(os.homedir(), 'micromamba/pkgs/'))
-      await execute('source ' + profile + ' && micromamba create --strict-channel-priority -y -f ' + envFilePath)
+      await execute('source ' + profile + ' && micromamba create -n ' + envName + ' --strict-channel-priority -y -f ' + envFilePath)
       fs.appendFileSync(profile, 'set -eo pipefail\n')
```

Comment:
hmm, is this manually edited? I think you need to edit the `index.js` that's in the root folder, and then build with nodejs (can get it from conda-forge) usign 

```
mamba install nodejs -c conda-forge
npm install
npm run build
```

Reply:
Ah yeah, I had both files open and didn't look well. What should I do with changes to `package-lock.json`?

Reply:
Commit those changes.